### DISCUSSION TITLE
E-mail Q&A no. 1

### DISCUSSION THREAD

**Discussion by fpetric** [larics/icuas24_competition#22] [General]
Couple of questions from the emails for everyone to see:

> How big is the environment room i.e. how much UAV movement space is allowed away from the beds?

The UAV is allowed to fly 3m away from the beds. This results in the box of size 20x27 meters. The height limit for the UAV is 8 meters.

> Is the UAV allowed to move above the topmost beds (to move across rows of beds)?

No. 

> Is the UAV allowed to move across rows, from the space between plants beds, where plants are absent?

Also no. Even if there are no plants in the bed, the UAV cannot fly over it. 

> Where and how to submit the intermediate submission of our work?

The submission should be in form of a docker image or a dockerfile, it should contain your copy of this repo with your session file that launches your code. More details will be available soon (tomorrowish, depending on the timezone). The deadline may be flexible a bit.

> What the intermediate submission should encompass?

You will need to write a simple node that sends the UAV to a couple of waypoints in the empty world. More details will be available soon, alongside the method for submission.

**Comment by joaovmamute** [comment]
Greetings,

What is the time limit for the task?

**Reply by fpetric** [reply]
For the intermediate task, no limit :)

For the real task, not sure at the moment. We are thinking about 10 minutes, but are open to adjusting depending on what the teams achieve.
